.changeset/many-emus-wear.md

```diff
@@ -0,0 +1,11 @@
+---
+'sku': minor
+---
+
+Vite and Vitest can be used without experimental flags
+
+The Vite bundler and Vitest test runner now have stable support and can be used in production.
```

Comment:
This statement isn't limited to Static Apps. Which might make people think it's supported for SSR. Should it not?